### PR TITLE
Svsi

### PR DESCRIPTION
**Pull request by pkheav**

### CODE REVIEW

**Comment by jeremyw24** [comment]
@stakach can you please review this PR?

File: drivers/amx/svsi/n_series_decoder.cr [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  include PlaceOS::Driver::Interface::InputSelection(Int32)
+
+  tcp_port 50002
+  descriptive_name "AMX SVSI N-Series Decoder"
+  generic_name :Decoder
+
+  @previous_stream : Int32? = nil
+  @mute : Bool = false
+  @stream : Int32? = nil
+
```
Comment (caspiano) [inline drivers/amx/svsi/n_series_decoder.cr:17]:
```suggestion

  private SEPARATOR = '\r'

```

File: drivers/amx/svsi/n_series_decoder.cr [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    do_send("setSettings", "wallStretch", scaling_mode)
+  end
+
+  def received(data, task)
+    data = String.new(data)
+    logger.debug { "Received: #{data}" }
+
+    prop, value = data.split(':')
+
+    case prop.downcase
```
[the remark this answers is not in the document]
Comment (caspiano) [reply drivers/amx/svsi/n_series_decoder.cr:149]: 100% good use-case for https://github.com/spider-gazelle/inactive-support/#mapped_enum

[the remark this answers is not in the document]
Comment (pkheav) [reply drivers/amx/svsi/n_series_decoder.cr:150]: yeah good idea, will update the driver to use that now that it's available

**Comment by jeremyw24** [comment]
@caspiano bump - be good to close out this PR.

**Pull request merged**